plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiTableProperties.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.hudi;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.session.PropertyMetadata;
+
+import javax.inject.Inject;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static io.trino.spi.session.PropertyMetadata.stringProperty;
+
+public class HudiTableProperties
+{
+    public static final String LOCATION_PROPERTY = "location";
+
+    private final List<PropertyMetadata<?>> tableProperties;
+
+    @Inject
+    public HudiTableProperties()
+    {
+        tableProperties = ImmutableList.<PropertyMetadata<?>>builder()
+                .add(stringProperty(
+                        LOCATION_PROPERTY,
+                        "File system location URI for the table",
+                        null,
+                        false))
+                .build();
+    }
+
```

Comment:
Is `format` maybe relevant ? Parquet / orc ?

Reply:
For the first phase, the connector will only support parquet format. Then we will add for ORC. https://issues.apache.org/jira/browse/HUDI-2694